client/my-sites/migrate/section-migrate.jsx

```diff
@@ -115,6 +115,15 @@ class SectionMigrate extends Component {
 		this.props.navigateToSelectedSourceSite( this.state.selectedSiteSlug );
 	};
 
+	requestMigrationReset = async ( targetSiteId ) => {
+		return await wpcom.req
```

Comment:
The `return` statement makes the `async`/`await` pointless: we can as well return the promise directly. It will return the POST response.

My idea was to ignore the POST response and always return `Promise<void>`.

Reply:
Lol, I'm pretty sure I removed the `return` but I guess I left it unintentionally. Removed in 57eb59d7d5d91098710ce1d46a2fcaae9c871380